woothemes-sensei-certificates.php

```diff
@@ -6,439 +6,28 @@
  * Version: 1.1.1
  * Author: Automattic
  * Author URI: https://automattic.com
+ * Requires PHP: 5.6
  * Woo: 247548:625ee5fe1bf36b4c741ab07507ba2ffd
  * License: GPLv2+
  */
 
-/**
- * TABLE OF CONTENTS
- *
- * - Required functions
- * - Plugin Updates
- * - Actions and Filters
- * - init_certificates_textdomain()
- * - init_sensei_certificates()
- * - sensei_certificates_install()
- * - sensei_certificates_updates_list()
- * - sensei_update_users_certificate_data()
- * - sensei_create_master_certificate_template()
- */
-
-require_once dirname( __FILE__ ) . '/classes/class-woothemes-sensei-certificates-dependency-checker.php';
-
-if ( ! Woothemes_Sensei_Certificates_Dependency_Checker::are_dependencies_met() ) {
-	return;
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
 }
 
-/**
- * Actions and Filters
- */
-add_action( 'plugins_loaded', 'init_certificates_textdomain' );
-add_action( 'plugins_loaded', 'init_sensei_certificates', 0 );
-add_filter( 'sensei_upgrade_functions', 'sensei_certificates_updates_list', 10, 1 );
-register_activation_hook( __FILE__, 'sensei_certificates_install' );
-
-
-/**
- * init_certificates_textdomain localization
- *
- * @since  1.0.0
- * @return void
- */
-function init_certificates_textdomain() {
-	load_plugin_textdomain( 'sensei-certificates', false, dirname( plugin_basename( __FILE__ ) ) . '/lang/' );
-} // End init_certificates_textdomain()
-
-
-/**
- * init_sensei_certificates function.
- *
- * @since  1.0.0
- * @return void
- */
-function init_sensei_certificates() {
-	require_once 'classes/class-woothemes-sensei-certificates-utils.php';
-	require_once 'classes/class-woothemes-sensei-certificates.php';
-	$GLOBALS['woothemes_sensei_certificates'] = new WooThemes_Sensei_Certificates( __FILE__ );
-	require_once 'classes/class-woothemes-sensei-certificate-templates.php';
-	$GLOBALS['woothemes_sensei_certificate_templates'] = new WooThemes_Sensei_Certificate_Templates( __FILE__ );
-	require_once 'classes/class-woothemes-sensei-certificates-data-store.php';
-
-} // End init_sensei_extension()
-
-
-/**
- * install function to generate cert hashes
- *
- * @since  1.0.0
- * @return string
- */
-function sensei_certificates_install() {
-
-	// Check if the installer has already been run
-	$sensei_certificates_user_data_installed = get_option( 'sensei_certificate_user_data_installer', false );
-	$sensei_certificate_templates_installed  = get_option( 'sensei_certificate_templates_installer', false );
-	$user_data_installed                     = false;
-	$template_installed                      = false;
+define( 'SENSEI_CERTIFICATES_VERSION', '2.0.0' );
```

Comment:
This should probs be 1.1.1 for the purpose of this PR, but we can change it when we release beta.